### ISSUE TITLE
How to record histrogram of nueron network when one of its layer is set to use pre-trained weights?

### ISSUE THREAD

**Issue by bkktimber** [awslabs/mxboard#34]
I followed this gluon's [document classification tutorial](https://gluon-nlp.mxnet.io/master/examples/sentence_embedding/self_attentive_sentence_embedding.html) and tried to visualise my training with mxboard. So far I can visualise my model architecture,  training loss, and accuracy.

However, my problem is when I want to record histograms of my model. Python throws this error:

`Runtime Error: "Cannot get gradient array for Parameter [my layer name] because grad_req='null'" `

then I checked my model initialise block and found these lines:
```
model.embedding_layer.weight.set_data(vocab.embedding.idx_to_vec)
model.embedding_layer.collect_params().setattr('grad_req', 'null')
```
and mxnet didn't run the gradient for this layer because I want to use my pre-trained embedding.

So, my question is how do I get gradients for the other layers except for the embedding layer?

**Comment by bkktimber** [comment]
this might be too late. but I found the bug in my code. 
thanks

**Comment by reminisce** [comment]
Thanks. This is issue is closed.
